config/environments/development.rb

```diff
@@ -4,27 +4,47 @@
 
   # Settings specified here will take precedence over those in config/application.rb.
 
+  benchmarking = ENV['BENCHMARKING'] == 'true'
```

Comment:
Logging is surprisingly expensive, and class reloading will obviously slow things down. This lets us easily flip a flag to get things a little closer to production.